engine/xml-config/src/test/java/org/datacleaner/job/JaxbJobWriterTest.java

```diff
@@ -365,6 +368,19 @@ public void testReadAndWriteOutputDataStreamsJob() throws Exception {
         assertMatchesBenchmark(job, "JaxbJobWriterTest-testReadAndWriteOutputDataStreamsJob.xml");
     }
 
+    public void testWriteVariable() throws Exception {
+        final DescriptorProvider descriptorProvider = new ClasspathScanDescriptorProvider().scanPackage(
+                "org.datacleaner", true);
+        CsvDatastore datastore = new CsvDatastore("date-datastore", "src/test/resources/example-dates.csv");
+        DataCleanerConfiguration configuration = new DataCleanerConfigurationImpl().withDatastores(datastore)
+                .withEnvironment(new DataCleanerEnvironmentImpl().withDescriptorProvider(descriptorProvider));
+        JaxbJobReader reader = new JaxbJobReader(configuration);
+        File file = new File("src/test/resources/example-job-variables.xml");
+        AnalysisJobBuilder ajb = reader.create(file);
+
```

Comment:
There was already a test in the JaxbJobReaderTest.testReadVariable. This one compares the output of the write with the benchmark and it needs to have the variable written back.